.github/workflows/sonarcloud.yml

```diff
@@ -60,12 +60,12 @@ jobs:
       run: cmake -DCMAKE_BUILD_TYPE=Profile -DSTRIP_SYMBOLS=ON source
     - name: Build
       run: build-wrapper-linux-x86/build-wrapper-linux-x86-64 --out-dir build-wrapper-output-dir make -j $(nproc)
-    - name: Tests python2
+    - name: Tests python3
       run: |
         export JORMUNGANDR_BROKER_URL='amqp://guest:guest@rabbitmq:5672//'
         export KRAKEN_RABBITMQ_HOST='rabbitmq'
         ctest --output-on-failure
-    - name: docker_test python2
+    - name: docker_test python3
```

Comment:
in docker_test, there're tests that are related to DB/tyr. which are still running on python2.7